web/src/components/FilingsCalendarTaxAccess.test.tsx

```diff
@@ -0,0 +1,338 @@
+import { getMergedConfig } from "@/contexts/configContext";
+import * as api from "@/lib/api-client/apiClient";
+import { generateProfileData, generateUserData, randomLegalStructure } from "@/test/factories";
+import { markdownToText, randomElementFromArray } from "@/test/helpers";
+import {
+  currentUserData,
+  setupStatefulUserDataContext,
+  WithStatefulUserData,
+} from "@/test/mock/withStatefulUserData";
+import { FormationLegalType } from "@businessnjgovnavigator/shared/index";
+import { randomInt } from "@businessnjgovnavigator/shared/intHelpers";
+import { OperatingPhases } from "@businessnjgovnavigator/shared/operatingPhase";
+import { UserData } from "@businessnjgovnavigator/shared/userData";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import { BusinessPersona } from "../../../shared/src/profileData";
+import { generateFormationData } from "../../test/factories";
+import { FilingsCalendarTaxAccess } from "./FilingsCalendarTaxAccess";
+
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/api-client/apiClient", () => ({
+  postTaxRegistrationOnboarding: jest.fn(),
+}));
+const mockApi = api as jest.Mocked<typeof api>;
+
+const Config = getMergedConfig();
+
+const renderFilingsCalendarTaxAccess = (initialUserData?: UserData) => {
+  render(
+    <WithStatefulUserData initialUserData={initialUserData}>
+      <FilingsCalendarTaxAccess />
+    </WithStatefulUserData>
+  );
+};
+
+describe("<FilingsCalendarTaxAccess />", () => {
+  let userData: UserData;
+  let flow: Exclude<BusinessPersona, undefined>;
```

Comment:
why exclude here?

Reply:
Because otherwise it's annoying to re-use flow in all the config lookups below